cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc

```diff
@@ -21,6 +21,11 @@ It may still throw an error if conflicting data, indexes, or constraints exist.
 Examples of this are nodes with missing properties, indexes with the same name, or constraints with same schema but a different constraint type.
 
 For constraints that are backed by an index, the index provider and configuration for the backing index can be specified using the `OPTIONS` clause.
+Valid values for the index provider is `native-btree-1.0`, `lucene+native-3.0`, and `range-1.0`, default if nothing is specified is `native-btree-1.0`.
+The index type of the backing index is set depending on the provider, the `range-1.0` generates a range index while the other providers generates a b-tree index.
+Valid b-tree configuration settings are `spatial.cartesian.min`, `spatial.cartesian.max`, `spatial.cartesian-3d.min`, `spatial.cartesian-3d.max`,
```

Comment:
With the number of options, it would probably look clearer with a bullet list, or a table. This could include the options for the index provider as well.